core/src/main/java/org/openedx/core/ui/ComposeCommon.kt

```diff
@@ -1045,40 +1044,48 @@ fun OfflineModeDialog(
         Row(
             Modifier
                 .fillMaxWidth()
-                .padding(16.dp),
+                .padding(vertical = 10.dp, horizontal = 24.dp),
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            Text(
-                modifier = Modifier.testTag("txt_offline_label"),
+            IconText(
                 text = stringResource(id = R.string.core_offline),
-                style = MaterialTheme.appTypography.labelMedium,
-                color = MaterialTheme.appColors.textDark
+                painter = painterResource(id = R.drawable.core_ic_offline),
+                color = Color.Black,
+                textStyle = MaterialTheme.appTypography.titleSmall
             )
-            Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
-                Text(
-                    modifier = Modifier
-                        .testTag("txt_dismiss")
-                        .clickable { onDismissCLick() },
-                    text = stringResource(id = R.string.core_dismiss),
-                    style = MaterialTheme.appTypography.labelMedium,
-                    color = MaterialTheme.appColors.primary
-                )
-                Text(
-                    modifier = Modifier
-                        .testTag("txt_reload")
-                        .clickable { onReloadClick() },
-                    text = stringResource(id = R.string.core_reload),
-                    style = MaterialTheme.appTypography.labelMedium,
-                    color = MaterialTheme.appColors.primary
-                )
+            Row(horizontalArrangement = Arrangement.spacedBy(36.dp)) {
+                IconButton(
+                    modifier = Modifier.size(20.dp),
+                    onClick = {
+                        onReloadClick()
+                    }) {
+                    Icon(
+                        modifier = Modifier.size(20.dp),
+                        painter = painterResource(R.drawable.core_ic_reload),
+                        contentDescription = null,
+                        tint = MaterialTheme.appColors.primary
+                    )
+                }
+                IconButton(
+                    modifier = Modifier.size(20.dp),
+                    onClick = {
+                        onDismissCLick()
+                    }) {
+                    Icon(
+                        modifier = Modifier.size(20.dp),
+                        painter = painterResource(R.drawable.core_ic_close),
```

Comment:
we can use `imageVector = Icons.Filled.Close,` instead, and remove this drawable.